graphs/frag_graph.py

```diff
@@ -139,6 +118,21 @@ def construct_vcf_for_frag_graph(self, input_vcf, output_vcf, vcf_dict):
         else:
             warnings.warn(output_vcf + ' already exists!')
 
+    def set_graph_properties(self, features, config=None):
```

Comment:
keep the original implementation since this is skipping all features other than betweenness